### PR TITLE
Mobile ui debug

### PR DESCRIPTION
**Pull request by eoghanTadhg**
Mobile cleanup UI and menu improvements.

### CODE REVIEW

File: packages/thriver-core-components/lib/component-sidebar/aside.js [hunk 1]
```diff
@@ -179,8 +179,8 @@ Template.aside.onRendered(function () {
                 tabs[i].click();
 
         // Otherwise, if there isn't any path, click the first tab for the user
-        if (!path.length && tabs.length)
-            tabs[0].click();
+        //if (!path.length && tabs.length)
+            //tabs[0].click();
```
Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-sidebar/aside.js]: Don't commit commented code.

Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-sidebar/aside.js]: Keep this code, but add a third condition on line 182 for window.width so the link doesn't get clicked on mobile.

File: packages/thriver-accounts/lib/templates/profile.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 		{{/each}}
         <input type="submit" value="{{submitValue}}">-->
 
-        {{> quickForm collection="Meteor.users" id="updateProfile" doc=doc type="update" 
-            omitFields=omitFields fields="profile" }}
+        {{> quickForm collection="Meteor.users" id="updateProfile" doc=doc type="update"
+            omitFields=omitFields fields="profile" template="series" }}
```
Comment (micchickenburger) [inline packages/thriver-accounts/lib/templates/profile.html]: You should either refrain from committing this part until templating is completed, or move these changes into a separate feature branch.

File: packages/thriver-core-components/lib/component-header/mobile-menu.less [hunk 3]
```diff
@@ -15,8 +15,9 @@
         span{pointer-events: none;}
         .shadowNone;
         &:active #nav-icon{
-            box-shadow:inset 0px 0px 100px 100px rgba(0,0,0,0.5);
-            -webkit-box-shadow:inset 0px 0px 100px 100px rgba(0,0,0,0.5);
+            //box-shadow:inset 0px 0px 100px 100px rgba(0,0,0,0.5);
+            //-webkit-box-shadow:inset 0px 0px 100px 100px rgba(0,0,0,0.5);
```
Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-header/mobile-menu.less:19]: Don't commit commented code.

File: packages/thriver-core-components/lib/component-tabs/tabs.js [hunk 4]
```diff
@@ -16,16 +16,16 @@ var toggleTabs = function (event) {
...
 	for (let i = 0; i < sections.length; ++i)
 		Thriver.util.hide(sections[i], true);
-        
+
     // Set link as active
     Thriver.util.makeActive(event.target, true);
 
     // Set section as active
-    Thriver.util.hide( menu.parentElement.querySelector('article[data-id="' + 
+    Thriver.util.hide( menu.parentElement.querySelector('article[data-id="' +
```
Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-tabs/tabs.js:28]: We should use ECMAScript 6 templates now :-D

```
Thriver.util.hide(menu.parentElement.querySelector(`article[data-id="${event.target.dataset.id}"]`), false);
```

File: packages/thriver-core-components/lib/component-tabs/tabs.js [hunk 5]
```diff
@@ -47,6 +47,25 @@ Template.body.events({
  * @method
  */
 Template.tabs.onRendered(function () {
+
+    //Deselect all active tabs if on mobile
+    if (window.innerWidth < 768){
```
Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-tabs/tabs.js:52]: Space required between `)` and `{`.

File: packages/thriver-core-components/lib/component-tabs/tabs.js [hunk 6]
```diff
@@ -47,6 +47,25 @@ Template.body.events({
...
+            tabs       = document.querySelectorAll('div.tabs > article');
+
+        // Remove expanded from all tab anchors
+        for (let i = 0; i < tabAnchors.length; ++i)
+            Thriver.util.makeActive(tabAnchors[i], false);
+
+        // Add hidden state to all tabs
+        for (let i = 0; i < tabs.length; ++i)
+            Thriver.util.hide(tabs[i], true);
+
```
Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-tabs/tabs.js:65]: Remove empty line

File: packages/thriver-core-components/lib/component-tabs/tabs.js [hunk 7]
```diff
@@ -47,6 +47,25 @@ Template.body.events({
...
+        // Tabs vars
+        var tabAnchors = document.querySelectorAll('menu.tabs > li > a'),
+            tabs       = document.querySelectorAll('div.tabs > article');
+
+        // Remove expanded from all tab anchors
+        for (let i = 0; i < tabAnchors.length; ++i)
+            Thriver.util.makeActive(tabAnchors[i], false);
+
+        // Add hidden state to all tabs
+        for (let i = 0; i < tabs.length; ++i)
```
Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-tabs/tabs.js:63]: `++i` and `i++` should now be `i += 1` per the airbnb style guide.

Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-tabs/tabs.js:63]: Loops and conditionals must use curly braces unless their only statement exists on the same line, per the airbnb style guide.

File: packages/thriver-core-components/lib/component-tabs/tabs.js [hunk 8]
```diff
@@ -47,6 +47,25 @@ Template.body.events({
...
+            Thriver.util.hide(tabs[i], true);
+
+    }
+
+    // I don't believe this is doing anything'
```
Comment (micchickenburger) [inline packages/thriver-core-components/lib/component-tabs/tabs.js:68]: What isn't?  Why not?

File: packages/thriver-core-styles/lib/styles/type.less [hunk 9]
```diff
@@ -48,6 +48,10 @@ h2{.font(2.4,2.8,0.025,@bold,none);}
 h3{.font(1.8,2.4,0.08,@normal,none);}
 h4{.font(1.6,1.6,0.1,@normal,none);}
 h5{.font(1.6,1.6,0,@bold,none);}
+
+@media @mobile{
+    main h2{.font(2.0,2.5,0.025,@bold,none) !important; padding-bottom: 5px !important;}
```
Comment (micchickenburger) [inline packages/thriver-core-styles/lib/styles/type.less:53]: Is `!important` necessary?

File: packages/thriver-core/lib/client/canvas.js [hunk 10]
```diff
@@ -112,35 +112,69 @@ Thriver.canvas = {
...
 
-			// Allow scrolling
-			document.body.classList.remove('noScroll');
+				// Remove Expanded From Toggle
+				Thriver.util.makeActive( document.getElementById("mobile-toggle"), false );
 
-			// Hide menu
-			Thriver.util.hide( document.getElementById('mobile-navigation'), true );
+				// We're done
+				return false;
```
Comment (micchickenburger) [inline packages/thriver-core/lib/client/canvas.js:153]: Tabs are not permitted... Please replace them with spaces for lines 149-153.

File: packages/thriver-core/lib/client/canvas.js [hunk 11]
```diff
@@ -112,35 +112,69 @@ Thriver.canvas = {
...
 
-			// We're done
-			return false;
+			}
 		}
 
 		// Open menu
 
+		// Set Toggle to Expanded
+		Thriver.util.makeActive( document.getElementById("mobile-toggle"), true );
```
Comment (micchickenburger) [inline packages/thriver-core/lib/client/canvas.js:161]: Double quotes not permitted in JavaScript files.  Please use single quotes.

File: packages/thriver-core/lib/client/canvas.js [hunk 12]
```diff
@@ -112,35 +112,69 @@ Thriver.canvas = {
...
 
+
+
+
+
+
+
+
+
+
```
Comment (micchickenburger) [inline packages/thriver-core/lib/client/canvas.js:177]: Remove all these empty lines.

File: packages/thriver-core/lib/client/scroll.js [hunk 13]
```diff
@@ -94,7 +94,8 @@ handleHeaderStateChange = function (event) {
         if ($(document).scrollTop() > 1000 && $(window).width() > 767)
             document.getElementById("back-to-top").classList.add('active');
         else
-            document.getElementById("back-to-top").classList.remove('active');
+            if(document.getElementById("back-to-top"))
```
Comment (micchickenburger) [inline packages/thriver-core/lib/client/scroll.js:97]: This should be moved to a higher level to also catch line 95.

File: packages/thriver-core/lib/client/scroll.js [hunk 14]
```diff
@@ -94,7 +94,8 @@ handleHeaderStateChange = function (event) {
...
+                document.getElementById("back-to-top").classList.remove('active');
```
Comment (micchickenburger) [inline packages/thriver-core/lib/client/scroll.js:98]: Double quotes not permitted.

File: packages/thriver-navigation/lib/templates/utility.html [hunk 15]
```diff
@@ -25,7 +25,7 @@
...
     <li class="{{#if equals position 'right'}}right{{/if}} {{#if more}}more{{/if}}">
-        <a href="{{#if equals type 'link'}}{{{url}}}{{/if}}{{#if equals type 'sidebar'}}/{{{sidebar}}}{{/if}}{{#if equals type 'null'}}/{{/if}}" target="{{#if equals type 'external'}}_blank{{/if}}" class="{{#if icon}}icon icon-{{{icon}}}{{/if}} {{#if iconAfter}}icon-after{{/if}} {{#if this.alerts}}{{#if show}}notificationAlert{{/if}}{{/if}}" aria-controls="{{#if equals type 'sidebar'}}{{{sidebar}}}{{/if}}" aria-expanded="false" data-toggle="{{#if equals type 'sidebar'}}canvas{{/if}}" data-action="{{#if action}}{{action}}{{/if}}">
+        <a href="{{#if equals type 'link'}}{{{url}}}{{/if}}{{#if equals type 'sidebar'}}/{{{sidebar}}}{{/if}}{{#if equals type 'null'}}/{{/if}}" target="{{#if equals type 'external'}}_blank{{/if}}" class="{{#if icon}}icon icon-{{{icon}}}{{/if}} {{#if iconAfter}}icon-after{{/if}} {{#if this.alerts}}{{#if show}}notificationAlert{{/if}}{{/if}}" aria-controls="{{#if equals type 'sidebar'}}{{{sidebar}}}{{/if}}" aria-expanded="false" data-toggle="{{#if equals type 'sidebar'}}canvas{{/if}}" data-action="{{#if action}}{{action}}{{/if}}" data-notification={{this.alert.show}}>
```
Comment (micchickenburger) [inline packages/thriver-navigation/lib/templates/utility.html:28]: Can this be made cleaner?

File: packages/wcasa:work/lib/client/work.js [hunk 16]
```diff
@@ -70,6 +70,10 @@ var changeTabs = function (event) {
             });
         }, 250);
     }
+    //Scroll to top of work on change tabs
+    // TODO: Use smoothScroll
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:74]: New syntax for TODO statements:

`// TODO(eoghantadhg): Use smoothscroll`

File: packages/wcasa:work/lib/client/work.js [hunk 17]
```diff
@@ -174,6 +178,7 @@ Template.workNav.events({
     'click h2': changeTabs,
     'click li > ul > li > a': changeTabs,
     'click button.backToTopWork': function (event) {
+        // TODO: Use smoothScroll
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:181]: New TODO syntax

File: packages/wcasa:work/lib/client/work.js [hunk 18]
```diff
@@ -192,6 +197,11 @@ Template.workNav.events({
         setTimeout(function () {
             document.body.classList.remove('workActive', 'workReading', 'workFadeOut');
         }, 200);
+
+        //Scroll to top of work on change tabs
+        // TODO: Use smoothScroll
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:202]: New TODO syntax

File: packages/wcasa:work/lib/client/work.js [hunk 19]
```diff
@@ -192,6 +197,11 @@ Template.workNav.events({
         setTimeout(function () {
             document.body.classList.remove('workActive', 'workReading', 'workFadeOut');
         }, 200);
+
+        //Scroll to top of work on change tabs
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:201]: Spaces required between `//` and text, per airbnb style guide.

File: packages/wcasa:work/lib/client/work.js [hunk 20]
```diff
@@ -192,6 +197,11 @@ Template.workNav.events({
...
+        // TODO: Use smoothScroll
+        offset = $('[id="what-we-do"]').offset().top + 228;
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/client/work.js:203]: `what-we-do` is a dynamic ID... We can't target it.

File: packages/wcasa:work/lib/templates/work.less [hunk 21]
```diff
@@ -62,6 +66,7 @@ section.work{
     //.main .workNav > ul li.new-section a:hover { color: @orange; }
     //.main .workNav > ul > li.new-section h2 a { background: @orange; }
     .workNav .new-section{white-space: nowrap; a{color: @teal !important;}}
+    .workActive & .workNav .new-section{a{color: lighten(@teal,50%) !important;}}
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:72]: Is `!important` necessary?

File: packages/wcasa:work/lib/templates/work.less [hunk 22]
```diff
@@ -359,7 +391,7 @@ section.work .workNav > ul{
     li a:hover{text-decoration: underline;}
     //li h2 a{background-color: rgba(6,33,49,0.2) !important;color: @white !important;}
     li h2 a{color: @white !important;text-decoration: none !important;}
-
+    li h2{padding-bottom: 0px !important;}
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:397]: Is `!important necessary`?

File: packages/wcasa:work/lib/templates/work.less [hunk 23]
```diff
@@ -841,6 +877,9 @@ section.work .main.container{
                     font-size: 12px;
                 }
             }
+            @media @mobile{
+                margin-left: -65px !important;
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:884]: Is `!important` necessary?

File: packages/wcasa:work/lib/templates/work.less [hunk 24]
```diff
@@ -1224,9 +1267,10 @@ section.work{
 }
 
 #main > section.work{
-    .main{position: static !important;}
+    .main{position: relative;} //Helps position sticky work nav
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:1273]: Space required between `//` and comment

File: packages/wcasa:work/lib/templates/work.less [hunk 25]
```diff
@@ -1264,4 +1313,20 @@ section.work .workNav .backToTopWork + li.new-section {
...
+            display: inline;
+            &:after{display: none;}
+            color: lighten(@teal,50%) !important;
+            background: transparent;
+        }
+    }
+
+}
+
+@media @mobile{ .workActive section.work .workNav > ul > li.backToIndexWork a{display: block !important;} }
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:1334]: Is `!important` necessary?

File: packages/wcasa:work/lib/templates/work.less [hunk 26]
```diff
@@ -1264,4 +1313,20 @@ section.work .workNav .backToTopWork + li.new-section {
...
\ No newline at end of file
+    .workActive &{
+        padding: 12px 0px !important;
+        background:rgba(0,0,0,0.1) !important;
+        a{
+            text-align: left;
+            text-indent: 20px;
+            display: inline;
+            &:after{display: none;}
+            color: lighten(@teal,50%) !important;
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:1327]: Is `!important` necessary?

File: packages/wcasa:work/lib/templates/work.less [hunk 27]
```diff
@@ -1264,4 +1313,20 @@ section.work .workNav .backToTopWork + li.new-section {
         display: block;
         padding: 5px;
     }
-}
\ No newline at end of file
+    .workActive &{
+        padding: 12px 0px !important;
+        background:rgba(0,0,0,0.1) !important;
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:1321]: Is `!important` necessary?

File: packages/wcasa:work/lib/templates/work.less [hunk 28]
```diff
@@ -1264,4 +1313,20 @@ section.work .workNav .backToTopWork + li.new-section {
         display: block;
         padding: 5px;
     }
-}
\ No newline at end of file
+    .workActive &{
+        padding: 12px 0px !important;
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:1320]: Is `!important` necessary?

File: packages/wcasa:work/lib/templates/work.less [hunk 29]
```diff
@@ -1264,4 +1313,20 @@ section.work .workNav .backToTopWork + li.new-section {
...
+            &:after{display: none;}
+            color: lighten(@teal,50%) !important;
+            background: transparent;
+        }
+    }
+
+}
+
+@media @mobile{ .workActive section.work .workNav > ul > li.backToIndexWork a{display: block !important;} }
+.mobile-article-open.workActive.workReading section.work .workContent{ @media @mobile{height: auto !important;}}
```
Comment (micchickenburger) [inline packages/wcasa:work/lib/templates/work.less:1335]: Is `!important` necessary?

**Review by micchickenburger — DISMISSED** [review]

File: packages/thriver-core-components/lib/component-sidebar/aside.js (hunk 1 above)
Comment (eoghanTadhg) [reply packages/thriver-core-components/lib/component-sidebar/aside.js]: [mobileUIDebug 851733b]

Comment (eoghanTadhg) [reply packages/thriver-core-components/lib/component-sidebar/aside.js]: [mobileUIDebug 851733b]

**Pull request closed without merging**